CHANGELOG.md

```diff
@@ -1,3 +1,60 @@
+# 93.4.0
+
+This minor release adds a new Filled and Outlined TextFields that will eventually replace MDCTextField's filled and outlined controllers. Additionally, it fixes dynamic type issues for multiple components, and removes deprecated themers.
+
+## New features
+
+### TextFields
+
+* [Add filled textfields (#8674)](https://github.com/material-components/material-components-ios/commit/f2e5fe4fbd17735858b1897765ce6c4f55147cb9) (Andrew Overton)
+* [Add outlined textfield (#8705)](https://github.com/material-components/material-components-ios/commit/cfc699b1992c56e2e67cfc082e568863ac80da18) (Andrew Overton)
+
+## API changes
```

Comment:
"API Changes" is usually that autogenerated bit, but it seems none was included. I think you can just include these sections under "Component Changes" below.

Reply:
Agreed

Reply:
Done